seaclouds-catalog.bom

```diff
@@ -254,6 +258,8 @@ brooklyn.catalog:
               planner.port: $brooklyn:component("planner").config("planner.config.port")
             install.latch: $brooklyn:component("planner").attributeWhenReady("service.isUp")
           - type: seaclouds-planner
+            brooklyn.config: 
+               planner.config.optimizer.url: $brooklyn:formatString("http://%s:%s", component("optimizer-service").attributeWhenReady("host.address"), component("optimizer-service").attributeWhenReady("http.port"))
```

Comment:
The option you propose is fine with me. 

If there are not comments against it at the end of the day, your opinion will became law and I will implement it.